product/reports/310_Migration Readiness - Providers/010_VMware Environment Summary.yaml

```diff
@@ -0,0 +1,84 @@
+---
+title: VMware Environment Summary
+rpt_type: Custom
+priority: 10
+db: Host
+cols:
+- name
+- vmm_product
+- vmm_version
+- num_cpu
+- v_total_vms
+include:
+  ext_management_system:
+    columns:
+    - name
+    - api_version
+col_order:
+- ext_management_system.name
+- ext_management_system.api_version
+- name
+- vmm_product
+- vmm_version
+- num_cpu
+- v_total_vms
+headers:
+- Provider
+- vSphere Version
+- Hypervisor
+- Product
+- Version
+- CPU Sockets
+- VMs
+conditions: !ruby/object:MiqExpression
+  exp:
+    "=":
+      field: Host.ext_management_system-type
+      value: ManageIQ::Providers::Vmware::InfraManager
```

Comment:
think using a Vmware field for the `db: ` host would also be more efficient

Reply:
as above

Reply:
is it possible to put a test on `Host-type` instead of the join into ems? (not sure if that actually exists)